src/liballoc/tests/vec.rs

```diff
@@ -945,6 +945,105 @@ fn drain_filter_complex() {
     }
 }
 
+#[test]
+fn drain_filter_consumed_panic() {
```

Comment:
Could you add another attribute?
```
#[cfg(not(miri))] // Miri does not support catching panics
```
Same for the other new test.

Reply:
@RalfJung Updated.